_episodes_rmd/13-dplyr.Rmd

```diff
@@ -98,6 +98,18 @@ gets that from the previous pipe. **Fun Fact**: There is a good chance you have
 encountered pipes before in the shell. In R, a pipe symbol is `%>%` while in the
 shell it is `|` but the concept is the same!
 
+> ## Tip: Renaming dataframe columns in dplyr
+>
+> In Chapter 4 we covered how you can rename columns with base R by assigning a value to the output of the `names` function. Just like select, this is a bit cumbersome, but thankfully dplyr has a `rename` function. 
```

Comment:
For Carpentries R style, we prefer functions to be followed by parentheses for distinguishing them from other objects (e.g. `names()`, `select()`, and `rename()`)